ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -39,53 +39,6 @@ tests:
   commands: find . -type f -name "*.sh" | xargs --verbose --no-run-if-empty shellcheck
   container:
     from: shiftstack-scripts-image
```

Comment:
do we need to maintain this file if all the jobs are moved?

Reply:
Yes. This configuration file contains:
* the “promotion” stanza for building the container image containing the scripts
* the presubmit that runs shellcheck on the scripts